internal/sirius/digital_lpa.go

```diff
@@ -15,6 +15,7 @@ type DigitalLpa struct {
 	InvestigationCount int        `json:"investigationCount"`
 	TaskCount          int        `json:"taskCount"`
 	WarningCount       int        `json:"warningCount"`
+	ObjectionCount     int        `json:"objectionCount"`
```

Comment:
Should probably add this to `digital_lpa_test` too (which will block it until Sirius can fulfil the contract)